CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+> When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+So you want to be a part of "The Compose Revolution" and that's great. Here are some contribution opportunities, in order of difficulty.
+
+## Documentation & Tutorials
+
+**Difficulty** : Beginner
+
+Go through the docs and tutorials, following the steps in each one. Identify mistakes where things are out of date or don't align with the code or are confusing and submit fixes.
```

Comment:
Go through the docs and tutorials, and follow the steps. Identify mistakes and parts that are confusing, and submit fixes.

Reply:
That's better. Will update.

Reply:
@B1ggDave  Updated via https://github.com/JetBrains/compose-jb/pull/511/commits/59f6636402682a1f8825ccfc0a74c2bc0b8c3813